shoop/admin/modules/products/views/forms.py

```diff
@@ -186,7 +186,18 @@ def __init__(self, **kwargs):
             self.fields["shops"].initial = [default_shop]
 
         self.file_url = self.instance.url
-        self.thumbnail = None
+
+        if self.instance.pk and self.instance.file:
+            if isinstance(self.instance.file, Image):
+                thumbnail = self.instance.easy_thumbnails_thumbnailer.get_thumbnail({
```

Comment:
This should be wrapped in some sort of `try: except`; `get_thumbnail` can and will throw when it just can't deal with things.